### DISCUSSION TITLE
Pick&Place Environment

### DISCUSSION THREAD

**Discussion by KevinGalassi** [isaac-sim/IsaacLab#43] [Q&A]
Hi,
I'm building an Isaac gym environment for pick and place of deformable objects, I was wondering how you are thinking to implement it in orbit for your Cloth folding/Spreading environment.

My problem is that pre_physics_step correspond to a single simulation step, while in my case it should be the entire pick and place trajectory execution.

Am i missing something?

**Comment by Mayankm96** [comment]
I did not completely understand what you're trying to do here. Is the goal to use these primitives as abstracted action space for your environments?

**Reply by PARKBONG** [reply]
Hi, @Mayankm96, Is there any cloth or deformable-related task available?
I checked the [development plan](https://isaac-orbit.github.io/orbit/source/refs/roadmap.html#development-roadmap), and currently, it is not supported officially. 
I appreciate you letting me know if there is any pre-release version or beta participation.

thanks.

**Reply by KevinGalassi** [reply]
i would appreiate a pre-release as well. I'm having problem with my cloth manipulation task, the cloth is constantly slipping out of the gripper when fingers are closed and is clearly something that I'm missing but the API pf isaac sim/omniverse are not so clear in many aspect...
